### PR TITLE
Refactor map_state to lifecycle_state

### PR DESCRIPTION
**Pull request by serhiibets**

### CODE REVIEW

File: wayat/frontend/test/services/status/map_status_service_test.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ import 'package:mockito/annotations.dart';
 import 'package:mockito/mockito.dart';
```
Comment (capf4bbo) [inline wayat/frontend/test/services/status/map_status_service_test.dart:5]: The name of the should be changed

File: wayat/frontend/lib/services/status/lifecycle_service.dart [hunk 2]
```diff
@@ -2,16 +2,18 @@ import 'package:get_it/get_it.dart';
...
 import 'package:wayat/services/common/http_provider/http_provider.dart';
 
-class MapStatusService {
+class LifeCycleService {
   final HttpProvider httpProvider = GetIt.I.get<HttpProvider>();
 
   ///Send [POST] request to backend to [Open GoogleMap]
-  Future sendMapOpened() async {
-    await httpProvider.sendPostRequest(APIContract.updateMap, {"open": true});
+  Future notifyMapOpened() async {
```
Comment (capf4bbo) [inline wayat/frontend/lib/services/status/lifecycle_service.dart:9]: This 2 methods should be merged into one, called notifyLifeCycleState(bool openned) {....}

**Review by capf4bbo — CHANGES_REQUESTED** [review]

File: wayat/frontend/lib/services/status/lifecycle_service.dart (hunk 2 above)
Comment (serhiibets) [reply wayat/frontend/lib/services/status/lifecycle_service.dart]: Fix

File: wayat/frontend/test/services/status/map_status_service_test.dart (hunk 1 above)
Comment (serhiibets) [reply wayat/frontend/test/services/status/map_status_service_test.dart:5]: Fix

**Review by capf4bbo — APPROVED** [review]

**Pull request merged**